dev/faq/tidb-binlog.md

```diff
@@ -0,0 +1,107 @@
+
+- When a Pump node runs normally or is in the `paused` state, if you use the `update-pump` command to set the Pump state to `offline`, the Drainer node stops pulling the binlog data from the `offline` Pump. In this situation, the newest binlog cannot be replicated to the Drainer node, causing data inconsistency between upstream and downstream.
+- When a Drainer node runs normally, if you use the `update-drainer` command to set the Drainer state to `offline`, the newly started Pump node only notifies Drainer nodes in the `online` state. In this situation, the `offline` Drainer fails to pull the binlog data from the Pump node in time, causing data inconsistency between upstream and downstream.
+
+## When can I use the `update-pump` command in binlogctl to set the Pump state to `offline`?
+
+You can use the `update-pump` command to set the Pump state to `offline` in these situations:
+
+- When a Pump process is exited abnormally and the service cannot be recovered, the replication task is interrupted. If you want to recover the replication and accept some losses of binlog data, use the `update-pump` command to set the Pump state to `offline`. Then, the Drainer node stops pulling binlog from the Pump node and continue replicating data.
+- Some stale Pump nodes are left over from historical tasks. Their processes have been exited and their services are no longer needed. Then, use the `update-pump` command to set their state to `offline`.
+
+For other situations, use the `offline-pump` command to close the Pump service, which is the regular process.
+
+> **Warning:**
+>
+> Do not use the `update-pump` command unless you accept loss of binlog data and data inconsistency between upstream and downstream or unless you no longer need the binlog data stored in the Pump node.
+
+## Can I use the `update-pump` command to set the Pump state to `offline` if I want to close a Pump node that is exited and set to `paused`?
+
+When a Pump process is exited and the node is in the `paused` state, not all the binlog data in the node is consumed in its downstream Drainer node. Therefore, doing so might risk data inconsistency between upstream and downstream. In this situation, restart the Pump and use the `offline-pump` command to close the Pump node.
+
+## When can I use the `update-drainer` command in binlogctl to set the Drainer state to `offline`?
+
+Some stale Drainer nodes are left over from historical tasks. Their processes have been exited and their services are no longer needed. Then, use the `update-drainer` command to set their state to `offline`.
+
+## Can I use SQL operations such as `change pump` and `change drainer` to pause or close the Pump or Drainer service?
```

Comment:
these sqls is introduced in `dev/how-to/maintain/tidb-binlog.md`, I guess users may don't under it if put here